src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.utils.ts

```diff
@@ -127,31 +133,59 @@ export const compareActivityItem = (
         {},
       );
 
-      const updatedOptions = activityItem.responseValues.options.reduce(
+      const sortedCurrOptions = getSortedOptions(activityItem.responseValues.options);
+      const updatedOptions = sortedCurrOptions.reduce(
         (options: Record<string, ItemOption>, { id, text, value }) => {
           if (!options[id]) {
+            // If there is a new option in the new version, we should display them in the same order as in the builder
+            // To add a new option at the bottom of the axis, we have to shift the values by 1 in the previous ones
+            // And also update the values in the answers
+
+            updatedAnswers = updatedAnswers.map((item) => ({
```

Comment:
maybe it's good idea to make a function for
answers.map((item) => ({
              ...item,
              answer: {
                ...item.answer,
                value: isValue(item.answer.value) ? +item.answer.value! + 1 : item.answer.value,
              },
            }));
     and reuse below